src/test/java/seedu/application/logic/commands/CommandTestUtil.java

```diff
@@ -49,8 +49,8 @@ public class CommandTestUtil {
     public static final String POSITION_DESC_GOOGLE = " " + PREFIX_POSITION + VALID_POSITION_GOOGLE;
     public static final String POSITION_DESC_FACEBOOK = " " + PREFIX_POSITION + VALID_POSITION_FACEBOOK;
 
-    public static final String INVALID_COMPANY_DESC = " " + PREFIX_COMPANY + "James&"; // '&' not allowed in names
-    public static final String INVALID_CONTACT_DESC = " " + PREFIX_CONTACT + "911a"; // 'a' not allowed in phones
+    public static final String INVALID_COMPANY_DESC = " " + PREFIX_COMPANY + "James&"; // '&' not allowed in companies
```

Comment:
definitely nitpicking but maybe change to `Google&` to match application book? so other groups don't get to nitpick on our code when they review it :')

Reply:
Good idea